js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -109,6 +116,8 @@ export default function PaidAdsSetupSections( {
 
 	const initialValues = {
 		amount: paidAds.amount,
+		budget,
```

Comment:
@dsawardekar We are passing the `budget` and `budgetMin` values to the initialValues of the form which is not needed really since they are not meant to be editable and does not need to be part of the form logic.

Reply:
Thanks @dsawardekar . I see we are passing `budget` which has the following properties:
* country
* daily_budget

However looks like we don't need `country`. How about we pass daily_budget directly instead of having it within an object?